src/Scripting/Core/ScriptOptions.cs

```diff
@@ -36,7 +36,6 @@ private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
                 return ImmutableArray<MetadataReference>.Empty;
             }
 
-            // Provide similar surface to mscorlib (netstandard 2.0).
             // These references are resolved lazily. Keep in sync with list in core csi.rsp.
```

Comment:
Right, that makes sense, but this surface area doesn't really look like mscorlib in netstandard 2.0 at all.

---
In reply to: [110797870](https://github.com/dotnet/roslyn/pull/18471#discussion_r110797870) [](ancestors = 110797870)